src/app/contextBroker/contextBroker.cpp

```diff
@@ -819,6 +826,10 @@ PaArgument paArgs[] =
   { "*",      CEAAT,   CEAAT_COMPS_V1,       "",               badVerbGetPostOnly                                  }
 
 
+#define SIMPLE_API_V2                                                                                                \
```

Comment:
SIMPLE_API_V2 will be changed to CONVENIENCE_OPERATIONS_V2, following the older names (when merged with develop).

NTC